pkg/driver/controller_server.go

```diff
@@ -864,6 +865,23 @@ func (driver *Driver) controllerPublishVolume(
 	publishContext[targetNamesKey] = strings.Join(publishInfo.AccessInfo.BlockDeviceAccessInfo.TargetNames, ",")
 	publishContext[targetScopeKey] = requestedTargetScope
 	publishContext[lunIDKey] = strconv.Itoa(int(publishInfo.AccessInfo.BlockDeviceAccessInfo.LunID))
+
+	// Start of population of target array details
+	log.Tracef("\n PUBLISH INFO ::::: %v", publishInfo)
+	numberOfSecondaryBackends := len(publishInfo.AccessInfo.BlockDeviceAccessInfo.SecondaryBackendDetails.PeerArrayDetails)
```

Comment:
do we need the count, can we just check if SecondaryBackendDetails not being nil?

Reply:
Done